peer/peer.go

```diff
@@ -155,10 +158,66 @@ func IDHexDecode(s string) (ID, error) {
 }
 
 // IDHexEncode returns the hex-encoded multihash representation of the ID.
+//
+// Deprecated: Don't raw-hex encode peer IDs, use base16 CIDs.
 func IDHexEncode(id ID) string {
```

Comment:
We should _really_ remove these. But that's a breaking change and we can do that later.